include/grpc/support/time.h

```diff
@@ -47,7 +47,11 @@ extern "C" {
 
 typedef struct gpr_timespec {
     time_t tv_sec;
+#if defined(GPR_WIN32) && defined(GPR_ARCH_64)
```

Comment:
Because, there are many type casting problems concerning time codes if tv_nsec is a 32bit integer in Win64.And, tv_sec is 64bit in Win64, so I try to fit the same in case of tv_nsec.